### PR TITLE
add feature to enable additional nginx config

### PR DESCRIPTION
**Pull request by ChristianUlbrich**
This adds the ability to specify additional config files to be used by nginx. It is useful, for additional hosts behind the web-facing proxy (e.g. additional management interfaces).

Additional *.conf files can either be added during built or by using volumes. I've updated the README.md accordingly and tested the image in production. :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,10 +21,13 @@ MAINTAINER Evan Brown <evanbrown@google.com>
 
 RUN rm /etc/nginx/conf.d/*.conf
 
+RUN mkdir -p /etc/nginx/extra-conf.d
```
Comment (evandbrown) [inline Dockerfile:24]: Possible to also add this to the packer.json file?

**Comment by evandbrown** [comment]
This looks solid. Thanks, Christian. Can you pleas sign the [Google CLA](https://cla.developers.google.com)? Once that's done I can get this merged.

File: Dockerfile (hunk 1 above)
Comment (ChristianUlbrich) [inline Dockerfile:24]: I am not really sure, what you'd expect. I am not using packer myself (yet). Something like:

```
{
      "type": "shell",
      "inline": [
        "rm /etc/nginx/conf.d/*.conf",
        "mkdir -p /etc/nginx/extra-conf.d"
      ]
    }
```

Separate PR? Or this PR, don't force me to squash my commits. :)

**Comment by ChristianUlbrich** [comment]
@evandbrown I signed the CLA.

**Comment by evandbrown** [comment]
Thank you, @ChristianUlbrich!

**Pull request merged**